lib/Bridge/TolerantParser/SourceCodeFilesystem/ScfClassCompletor.php

```diff
@@ -87,6 +89,17 @@ public function complete(Node $node, string $source, int $offset): Response
         return Response::fromSuggestions($suggestions->sorted());
     }
 
+    private function isAlreadyImported(string $candidate, Node $node): bool
+    {
+        foreach ($node->getImportTablesForCurrentScope()[0] as $resolvedName) {
```

Comment:
This is quite inefficient, if there are 100 suggestions, it will build the import table 100 times. Would be better to extract the import table before the loop and pass it to this function I think?